Program.cs

```diff
@@ -1,10 +1,69 @@
-﻿namespace WeatherMonitoringSystem
+﻿using WeatherMonitoringSystem.model;
+using WeatherMonitoringSystem.Observer;
+using WeatherMonitoringSystem.Strategy;
+using WeatherMonitoringSystem.utils;
+
+namespace WeatherMonitoringSystem
 {
     internal class Program
```

Comment:
_I forgot to add this comment before_

Write `Program.cs` in top-level statements instead of `Program` class and `Main` method.